docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: "Introduction to the Dagster Tutorial and Assets | Dagster Docs"
+description: "Learn about the objectives for the Dagster tutorial and about assets"
+noindex: True
+---
+
+# Introduction to Dagster
+
+Dagster is an orchestrator that's designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
+
+In this tutorial, you will analyze activity on a popular news aggregation website. Similar to your data pipelines, you will fetch data from the website, clean it up, and build a report that summarizes some findings. You'll then tell Dagster to occassionally update the data and the report, which Dagster calls _assets_.
```

Comment:
In this tutorial, you will build a basic ETL pipeline to analyze activity on a popular news aggregation

Reply:
An earlier draft used the expression "ETL", but the example isn't truly ETL, so we opted for "data pipeline".

Reply:
I was the one that pushed for this. Happy to chat through it and get it resolved it helpful.